docs/UserGuide.md

```diff
@@ -3,38 +3,16 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
-
+Doc’it provides a centralised platform for authorised staff from small family clinics to view, update, and onboard
+patient records, solving the inefficient paper records and files used today. With Doc’it, small family clinics can
+reduce man-hours in managing paper files, translating this ‘saved’ time into better frontline care services.
 * Table of Contents
 {:toc}
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Quick start
-
```

Comment:
@huyuxin0429 Let's not delete all of the Quick Start section into ```to be completed``` because it provides us a base to work with.